distribution/docker/Dockerfile

```diff
@@ -65,19 +73,26 @@ ARG TARGETARCH
 # Although bash-static supports multiple platforms, but there's no need for us to support all those platform, amd64 and arm64 are enough.
 #
 ARG BASH_URL_BASE="https://github.com/robxu9/bash-static/releases/download/5.1.016-1.2.3"
-RUN if [ "$TARGETARCH" = "arm64" ]; then \
-      BASH_URL="${BASH_URL_BASE}/bash-linux-aarch64" ; \
-    elif [ "$TARGETARCH" = "amd64" ]; then \
-      BASH_URL="${BASH_URL_BASE}/bash-linux-x86_64" ; \
+RUN if [ "$BASE_IMAGE" = "gcr.io/distroless/java$JDK_VERSION-debian11" ]; then \
```

Comment:
again, instead of validating the base image, why not simply test to see if we can call bash. If it works that means we don't need to install it.

Reply:
So I think, I should add the installation of bash script in end, after all the conditionals are done right? And keep `if bash installed` the only check ?

Reply:
not sure what you mean by adding it at the end. What I mean is to replace the current if statement with a different one, e.g. `if [ -x "$(command -v bash)" ]` which tests if the command exists. At this point in the script, we either have busybox or some other shell installed.

Reply:
we can keep it where it is. subsequent steps no longer need to differentiate based on bash. You can instead differentiate based on the presence of the other commands (e.g. groupadd)